### PR TITLE
[9.0] document_page_health_safety_manual

### PR DESCRIPTION
**Pull request by naousse**

### CODE REVIEW

File: document_page_health_safety_manual/README.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
 Health and Safety Manual Template
 =================================
 
-This module was written to extend the functionality of 
+This module was written to extend the functionality of
```
Comment (elicoidal) [inline document_page_health_safety_manual/README.rst]: This module extends the functionality of

**Comment by max3903** [comment]
:+1:

File: document_page_health_safety_manual/data/document_page_health_safety_manual.xml [hunk 2]
```diff
@@ -1,15 +1,10 @@
...
-    <data>
-
+<odoo>
       <record id="document_page_health_safety_manual" model="document.page">
         <field name="name">Health and Safety Manual</field>
-        <field name="parent_id" ref="mgmtsystem_manuals.manuals"/>
-        <field name="type">content</field>
+        <field name="type">category</field>
         <field name="content">
 TODO: Add the structure of the health and safety manual from the OHSAS 18001 standard.
```
Comment (dreispt) [inline document_page_health_safety_manual/data/document_page_health_safety_manual.xml]: It's better to use `<data nopudate="1">`

**Comment by astirpe** [comment]
:+1:  code review and functional test

**Comment by max3903** [comment]
@astirpe Thank you very much!

**Pull request merged**